README.md

```diff
@@ -1 +1,19 @@
 # hw01
+## Sean Ennis Biography
+
+Hello Friends! My name is Sean Ennis and I am a third year **Public Policy Studies** major and am undecided if I want to pursue a **Joint BA** degree or a **Human Rights** minor _though I am running out of time to decide_.
+
+Some of my academic and non-academic interests are:
+
+* Academic
+  * Politics, I recommend [fivethirtyeight.com](https://fivethirtyeight.com)
+  * Domestic and International Economic Policy
+  * Human Rights Law
+* Personal
+  * Reality Competition Shows
+  * Podcasts
+  * Football
+
+I've been completely let down by _Big Brother 22_ this season, so I just want to give a shout-out to Queen Da'Vonne Rogers ![BB22 Da'Vonne](https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcRkYDO1oMoinwifbh8m8kajXJftZLVKKIO6Lg&usqp=CAU)
```

Comment:
Really liked the image, but it looked a little bit small on the actual document. Maybe find a larger one and upload it into your document, then added it with a relative link. And I think this is the only problem that I can point out, great work!

Reply:
More importantly, I suggest putting the image on a separate line rather than running into the sentence itself. Would be more clear formatting